lib/utils.js

```diff
@@ -788,3 +788,18 @@ exports.safeJsonStringify = function safeJsonStringify(json) {
     return vuvuzela.stringify(json);
   }
 };
+
+exports.parseDdocFunctionName = function (s) {
+  var parts = s.split('/');
+  if (parts.length === 2) {
+    return parts;
+  } else if (parts.length === 1) {
+    return [s, s];
+  } else {
+    throw ("Unrecognized function name");
```

Comment:
I would much rather be returning false here, we arent being super consistent about it but generally exceptions are exceptional, this is user input and can be expected to fail

Also as a style need there no need for the `} else`'s and I think they confuse the fact that it will early return